sentry-ruby/lib/sentry/utils/argument_checking_helper.rb

```diff
@@ -15,5 +15,11 @@ def check_argument_includes!(argument, values)
         raise ArgumentError, "expect the argument to be one of #{values.map(&:inspect).join(' or ')}, got #{argument.inspect}"
       end
     end
+
+    def check_callable!(name, value)
```

Comment:
Unless we expect users to pass a command object here (Non-Proc objects that define `call` as the unify interface), we can use `check_argument_type!(argument, ::Proc, NilClass)` without introducing a new method. (Which I should've done for Configuration a while ago).

Reply:
this method was just pulled out of configuration.rb, we can do that refactoring separately